Frameworks/include/UITouchInternal.h

```diff
@@ -17,29 +17,31 @@
 
 #import <UIKit/UITouch.h>
 
-@class WUIPointerPoint;
-@class WUXIPointerRoutedEventArgs;
+#include "COMIncludes.h"
+#import <winrt/Windows.UI.Input.h>
+#import <winrt/Windows.UI.Xaml.Input.h>
+#include "COMIncludes_End.h"
 
 @interface UITouch () {
 @public
     // Adding a WUXIPointerRoutedEventArgs property so that we can mark the routed event as handled/not handled
     // when touchsBegan:withEvent method is called.
-    StrongId<WUXIPointerRoutedEventArgs> _routedEventArgs;
+    TrivialDefaultConstructor<winrt::Windows::UI::Xaml::Input::PointerRoutedEventArgs> _routedEventArgs;
     NSTimeInterval _timestamp;
     float _touchX, _touchY;
     float _previousTouchX, _previousTouchY;
     unsigned _tapCount;
     UITouchPhase _phase;
     StrongId<UIView> _view;
-    WUIPointerPoint* _pointerPoint; // Only valid during the processing of a pointer event; don't try to retain it.
+    TrivialDefaultConstructor<winrt::Windows::UI::Input::PointerPoint> _pointerPoint;
```

Comment:
That is a good question, because it's not very obvious.

C++/WinRT objects have a default constructor that _instantiates the object_. It is not like ComPtr or other smart types where the default constructor leaves the object uncreated.

C++/WinRT also have a constructor that takes nullptr_t that leaves the object initialized to null, but fields in Objective-C++ can only be initialized with the default constructor, so the nullptr_t constructor is not available. Therefore I've created TrivialDefaultConstructor<> that wraps a C++/WinRT object and forwards the default constructor to the nullptr_t constructor.